.env.test

```
@@ -17,8 +17,8 @@ LANGUAGE_PREFERENCE_COOKIE_NAME=openedx-language-preference
 SITE_NAME=localhost
 MARKETING_SITE_BASE_URL=http://localhost:18000
 SUPPORT_URL=http://localhost:18000/support
-ORDER_HISTORY_URL=http://localhost:8140/orders/unified/order_history/
-RECEIPT_URL=http://localhost:18130/checkout/receipt/
+ORDER_HISTORY_URL=http://localhost:18130/api/v2/orders/
+RECEIPT_URL=http://localhost:18130/api/v2/checkout/receipt/
```

The legacy e-commerce receipt path is `/checkout/receipt/`
```suggestion
RECEIPT_URL=http://localhost:18130/checkout/receipt/
```

---

Everthing was based off of ECOMMERCE_BASE_URL... whihc had api/v2 added to it.. so it is needed here... (i tested in both modes, it works locally)